api/container/container.py

```diff
@@ -46,9 +53,17 @@ class Container(containers.DeclarativeContainer):
         news_service
     )
 
+    games_dal = providers.Factory(GameDAL, session=db_session)
+
```

Comment:
no need to have that many whitespaces

just do all the DALs -> whitespace -> all the services
or smth like that

cuz it looks like a zebra as it is now